nomad/eval_broker.go

```diff
@@ -274,6 +274,8 @@ func (b *EvalBroker) enqueueWaiting(eval *structs.Evaluation) {
 }
 
 // enqueueLocked is used to enqueue with the lock held
+//
+// Are all evaluations we're enqueueing at this point pending? <---
```

Comment:
Looks like this got left in from office hours 😁

Reply:
Ha, oh my. Thank you